app/_how-tos/synchronize-with-git.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Synchronize Insomnia content with Git
+
+products:
+- insomnia
+
+tags:
+- documents
+- collections
+- mock servers
+- environments
+- git
+
+prereqs:
+    inline:
+    - title: Git repository
+      content: |
+        Create an empty remote Git repository.
+      icon_url: /assets/icons/git.svg
+    - title: Insomnia workspace
+      content: |
+        Create a workspace in Insomnia. A workspace can be a [design document](/how-to/create-a-design-document), a [collection](), a [mock server](), or an [environment]().
+      icon_url: /assets/icons/git.svg
+
+tldr:
+    q: How can I push content from Insomnia to a Git repository?
+    a: Create an empty Git repository and an Insomnia workspace. In the workspace, click the button at the bottom of the left pane and connect the repository, then commit and push your content.
```

Comment:
Is this a local empty git repository or is it a new repository within the git provider?

Reply:
In the provider, you can't connect a local repo, I'll make that clearer